changelog/1.20.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: Added the ability to set a site-wide workspace template policy at
+  **Manage > Admin > Templates > Template Policy**. If not set, Coder uses the
+  provided default.
+- web: Added the _nodeSelectors_ field to workspace templates.
+- other: Added a new JSON schema for writing Coder workspace as code templates
+  with code completion and syntax checking.
+- web: Added a service message that's displayed to all users of the system. The
+  message can be dismissed by each user at any point and will not be shown again
+  until there is a new message.
+- web: Added text wrapping to system banners.
+- templates: Added Kubernetes annotations to workspace templates.
+- infra: Increased network performance for creating new workspaces.
+- infra: Added a `CODER_RUNTIME` environment variable that indicates whether a
+  workspace is CVM-enabled or not.
+- web: Updated UI to display decommissioned workspaces that are awaiting
+  deletion.
+
+### Bug fixes 🐛
+
+- web: Fixed bug causing duplicate fetch requests on page load.
+- web: Fixed issue causing private dev URLs to load as blank pages for
+  unauthorized users (these users now see an error message).
+
+### Security updates 🔐
+
+- web: Require administrative permissions to view and modify workspaces
+  belonging to other users; previously, users could view others' workspace
+  metadata
+- web: Added content security policy (CSP) to help protect against cross-site
+  scripting attacks.
+- web: Added opt-in for HTTP Strict Transport Security. This setting can be
+  managed at **Manage > Admin > Infrastructure > HTTP Strict Transport
+  Security**.
+- web: Added opt-in for secure cookies. This setting can be managed at
+  **Manage > Admin > Infrastructure > Secure Cookie**.
+- web: Use strong cryptographic APIs to generate random numbers in backend and
+  frontend.
+- infra: Upgraded control plane containers from Red Hat UBI 8.3 to 8.4, and
+  switch from ubi to ubi-minimal to reduce container contents.
+- infra: Enable read-only root filesystem for control plane containers, by
+  default. You can override this with the Helm `coderd.securityContext` setting.
+- web: Resolved CVE-2021-23364 in browserslist.
+- web: Resolved CVE-2021-23358 in underscore.
+- web: Resolved CVE-2020-7753 in trim.
+- web: Added ability to filter the audit log by the _auto-off_ action.
+- other: Added `gosec` as a required check in the Coder development process; all
+  flagged results were reviewed and addressed.
+- other: Added `SonarCloud` static code analysis as a required check in the
+  Coder development process; all flagged results were reviewed and addressed.
```

Comment:
I don't think this should be in the changelog, perhaps it could go on a security page describing how we do security

```suggestion
```